index.md

```diff
@@ -28,6 +28,9 @@ pip install --user --upgrade .
 Note: In order to report current state, installing nmstate as a non-root user
 should be enough. For change support, install the package as root.
 
+For installing stable release or other installation methods, please refer to
+[Nmstate Installation Guide](./install.md).
+
 ### Dependencies
 In order to support specific capabilities, additional packages are required:
```

Comment:
Don't wee need to add minimal NetworkManager version nmstate supports ? I am thinking on future vlan_filtering, so maybe even NetworkManager version per feature.

Reply:
Add:
```
Nmstate requires NetworkManager 1.12 or later version installed and started.
```